maintaining-and-tuning/monitoring.md

```diff
@@ -153,3 +153,25 @@ access to it can read aggregated metrics about your GRR installation. With these
 about the number of workers, flow activity, and service health can be derived. Make sure to limit
 access to the port, for example by employing a firewall. Furthermore, read
 [Prometheus Security](https://prometheus.io/docs/operating/security/).
+
+## Example visualization setup
+This example will walk you through setting up Grafana as a dedicated visualization software to parse, display, query and set up alerts for the metrics scraped by Prometheus. These instructions assume that GRR server and Prometheus are both up and running.
+
+1. [Install Grafana](https://grafana.com/docs/grafana/latest/installation/#install-grafana) as suggested to your operating system.
```

Comment:
I'd link the Ubuntu/Debian package right away: https://grafana.com/docs/grafana/latest/installation/debian/